source/Cosmos.System2/Graphics/Canvas.cs

```diff
@@ -277,7 +297,14 @@ public virtual void DrawPolygon(Pen pen, params Point[] points)
             DrawLine(pen, points[0], points[points.Length - 1]);
         }
 
-        public virtual void DrawRectangle(Pen pen, int x, int y, int width, int height)
+        public virtual void DrawRectangle(Pen pen, Point point, int width, int height = -1)
```

Comment:
The idea to have height with default value of -1 is to have the possibilty to do "DrawSquare()"? I'm unsure if it was not more clear to have an explicit DrawSquare() at that point.